src/click/core.py

```diff
@@ -2625,6 +2625,13 @@ class Option(Parameter):
     :param hidden: hide this option from help outputs.
     :param attrs: Other command arguments described in :class:`Parameter`.
 
+    .. versionchanged:: 8.3.0
+        Fixed a regression where options with ``flag_value`` and ``is_flag=False``
```

Comment:
This change comment is way too verbose. "Fix behavior when ``flag_value`` and ``is_flag`` are set."